Camille/gen/Enums/Champion.cs.dt

```diff
@@ -0,0 +1,40 @@
+﻿{{
+    const champions = require('./.champion.json')
+        .filter(({ id }) => id > 0)
+        .sort(({ name: a }, { name: b }) => a > b ? 1 : -1);
+    const hashFactor = 256;
+    const enumName = function(name) { return name.toUpperCase().replace(/[^A-Z]+/, '_'); };
+    const strHash = function(str) {
+        let h = 0;
+        for (let c of str)
+            h = hashFactor * h + c.charCodeAt(0);
+        return h;
+    };
+    const padId = function(id) { return ('' + id).padEnd(3); };
```

Comment:
We can remove these and hashFactor above unless you have a plan for them

(We will need a way to get the champion "key" from the champion enum though, since ddragon tends to use those over the ids. (i.e. "Wu Kong" is `MonkeyKing`), but that can be a separate PR)

Reply:
I will remove these functions and see if I can find a graceful solution for the champion key.